tests/global.spec.ts

```diff
@@ -1,11 +1,27 @@
 import { test, expect } from '@playwright/test';
 import {
+    compareVersion,
     matchesWithNodeName,
     siteMatch,
     slashNeededForUrl,
     trimURL
 } from '../keepassxc-browser/common/global.js';
 
+test('Test compareVersion()', async ({ page }) => {
```

Comment:
Need tests for keepass.compareVersions

Reply:
At this point functions in `keepass.js` cannot be tested due to different dependencies the file expects. One solution could be to move this function entirely to `global.js` and `keepass.js` just returns its value directly.

Reply:
Nah it's good as it is